src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Helper/Base64EncoderHelper.cs

```diff
@@ -0,0 +1,802 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+#if NETCOREAPP
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+#endif
+
+namespace System.Buffers.Text
+{
+    internal static partial class Base64Helper
+    {
+#pragma warning disable CA1805 // Member 's_base64ByteEncoder' is explicitly initialized to its default value
+        internal static Base64EncoderByte s_base64ByteEncoder = default;
```

Comment:
As commented earlier, all of these statically cached instances can be removed, and just use `default(T)` instead of reading this field.